### DISCUSSION TITLE
Removing patch to force popups into new tabs

### DISCUSSION THREAD

**Discussion by Eloston** [Eloston/ungoogled-chromium#1377] [Ideas]
Followup of https://github.com/Eloston/ungoogled-chromium/issues/38#issuecomment-771395015

Popup blocking is a common practice these days, and many websites have moved to HTML/CSS-based popups. Is it still necessary to force all popups into tabs? What do you guys think?

(I'm okay to remove this feature or move it behind a flag.)

**Comment by achesnok** [comment]
@Eloston I would like to have a toggle flag for this feature. Even in the simplest browser-wide implementation.

**Comment by kramred** [comment]
I'm voting for keeping the feature, and introducing a flag in case people want to toggle the behaviour

**Comment by Ahrotahn** [comment]
My initial reaction would be to say to keep the feature, but this post got me thinking about how often I've encountered popups and I wonder if I would notice if it were removed.  I guess I wouldn't be able to really know until I could turn it off for a while.  

After messing with it during the update I know what would need to be done to add the flag and don't mind doing the work if that's the route you want to take.

**Reply by Eloston** [reply]
That'd be great! Our users will certainly appreciate it :+1:

**Comment by wchen342** [comment]
There is one particular scenario that I encountered in the past (in Firefox, not in Chromium but I assume something similar can happen). So some websites has a `mailto:` link and if you click on it, it will open the default mail client. However, if you set the default mail client to your browser, then the browser will go into a loop of keeping opening the same `mailto:` link in new tabs infinitely until there are hundreads of blank tabs. I am not sure whether opening in a new window will solve the problem though.
